src/www/ui/api/Models/Analysis.php

```diff
@@ -141,13 +162,24 @@ public function setUsingArray($analysisArray, $version = ApiVersion::V1)
     if (array_key_exists("ojo", $analysisArray)) {
       $this->ojo = filter_var($analysisArray["ojo"], FILTER_VALIDATE_BOOLEAN);
     }
+    if (array_key_exists("scanoss", $analysisArray)) {
+      $this->scanoss = filter_var($analysisArray["scanoss"], FILTER_VALIDATE_BOOLEAN);
+    }
     if (array_key_exists("reso", $analysisArray)) {
       $this->reso = filter_var($analysisArray["reso"], FILTER_VALIDATE_BOOLEAN);
     }
     if (array_key_exists("package", $analysisArray)) {
       $this->pkgagent = filter_var($analysisArray["package"],
         FILTER_VALIDATE_BOOLEAN);
     }
+    if (array_key_exists("patent", $analysisArray)) {
+      $this->ipra = filter_var($analysisArray["patent"],
+        FILTER_VALIDATE_BOOLEAN);
+    }
+    if (array_key_exists("heritage", $analysisArray)) {
+      $this->softwareHeritage = filter_var($analysisArray["heritage"],
```

Comment:
```suggestion
    if (array_key_exists("ipra", $analysisArray)) {
      $this->ipra = filter_var($analysisArray["ipra"],
        FILTER_VALIDATE_BOOLEAN);
    }
    if (array_key_exists("softwareHeritage", $analysisArray)) {
      $this->softwareHeritage = filter_var($analysisArray["softwareHeritage"],
```

Please keep naming consistent.

Reply:
done